trailbot_bringup/logitech_camera/config/camera_info.yaml

```diff
@@ -0,0 +1,20 @@
+image_width: 1280
```

Comment:
Are these parameters accurate or do we parse the values from somewhere else?

Reply:
The values in camera_info.yaml are utilizied and set as such to publish an image to the /camera topic that is 1280x720 and raw as originally output from the camera. The output images were used to get the camera intrinsic matrix, and the true intrinsic matrix is applied in the trail detection and human detection nodes